common/libs/wallet/base.js

```diff
@@ -1,6 +1,6 @@
 // @flow
-import { stripHex } from 'libs/values';
-import type { RawTransaction } from 'libs/transaction';
+import { stripHex } from '../values';
+import type { RawTransaction } from '../transaction';
```

Comment:
@bfillmer @skubakdj @wbobeirne 

Open to suggests on adjust the `derivation-checker` command so that these relative imports aren't necessary.